docs/en/drivers/ios-xcuitest.md

```diff
@@ -99,6 +99,13 @@ following desired capabilities:
 * `udid` - the specific id of the device to test on. This can also be set to
    `auto` if there is only a single device, in which case Appium will determine
    the device id and use it.
+   
+### Optional Setup
+
+Although not strictly required, we recommend installing FBSimulatorControl for better
+results.
+
+`brew install FBSimulatorControl`
```

Comment:
not really:

```
# Get the Facebook Tap.
brew tap facebook/fb
# Install fbsimctl from master
brew install fbsimctl --HEAD
```

https://github.com/facebook/FBSimulatorControl/blob/master/fbsimctl/README.md

Reply:
Yeah, I didn't read the documentation :p